sites/funcorporate/src/backend/funcorporate/settings.py

```diff
@@ -327,13 +327,13 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         (
             "richie.apps.search.filter_definitions.IndexableMPTTFilterDefinition",
             {
-                "human_name": _("Subjects"),
+                "human_name": _("Training domains"),
```

Comment:
After bootstrap, this key was not translated but i18n files was up to date. I retyped manualy this key and `make i18n` again to see the translated message 🤷‍♂️

Reply:
Maybe it has to do with the fact that this is parsed upon container startup.

Reply:
Ok got it. It's because the compilemessages command is run in the dockerfile but in docker-compose, we override /app with our local code. So in your case, the .mo file (which is git ignored) was not present at the moment the container started. We should add a `make i18n` command in the bootstrap before `make run`.

Reply:
Thanks for explanation